src/lib/checkbox/checkbox.spec.ts

```diff
@@ -814,8 +814,8 @@ class SingleCheckbox {
   checkboxColor: string = 'primary';
   checkboxValue: string = 'single_checkbox';
 
-  onCheckboxClick(event: Event) {}
-  onCheckboxChange(event: MdCheckboxChange) {}
+  onCheckboxClick(_event: Event) {}
+  onCheckboxChange(_event: MdCheckboxChange) {}
```

Comment:
Should change these to be properties instead of methods since they're reassigned anyway, e.g.
```ts
onCheckboxClick: (Event?) => void = () => {};
```
(here and elsewhere that follows this pattern)